packages/model-viewer/src/test/helpers.ts

```diff
@@ -225,4 +225,58 @@ export const createFakeThreeGLTF = () => {
     } as unknown as GLTFParser,
     userData: {}
   };
-};
\ No newline at end of file
+};
+
```

Comment:
At first i put all the scenarios config into a json file and tried to fetch that in the suite()'s setup.
```
suite("fidelity test", ()=>{
let scenarios = [];
setup(()=>{
    scenarios = await(await fetch(scenarioPath)).json();
});

for(let scenario of scenarios){
    test(scenario.name, ()=>{
       //do the fidelity test here
    })
}
})
```
But the fact is that setup() only triggers before test(),  and when mocha see the line for(let scenario of scenarios), it didn't see test, so it doesn't call setup and scenarios is just empty.() I have tried some workaround, none work so far. So i just used this ugly inefficient way to read scenario configs. If you have better way to do this, im happy to learn